### DISCUSSION TITLE
"DOKAN_START_ERROR" when target directory is not empty

### DISCUSSION THREAD

**Discussion by Dwedit** [dokan-dev/dokany#1042] [General]
I was just testing out the Mirror application, and saw that it was displaying a confusing error message "Driver something wrong".  I figured out that it was a "DOKAN_START_ERROR" caused by specifying a target directory which was not empty.

Maybe it should be a DOKAN_MOUNT_ERROR instead?

**Comment by Liryna** [comment]
Hi @Dwedit ,

Thanks for the report! The V2 mount flag actually makes this now possible to see the different types of failures.
Done with https://github.com/dokan-dev/dokany/commit/d4de1bc616bffd284122fe2ec1adbfba7c4702ef
